### ISSUE TITLE
ESLint 6 and 7 are both required

### ISSUE THREAD

**Issue by Krinkle** [wikimedia/eslint-config-wikimedia#316]
```json
    "eslint": "^7.6.0",
    "eslint-config-wikimedia": "0.17.0",
```
> `npm WARN eslint-plugin-vue@6.2.2 requires a peer of eslint@^5.0.0 || ^6.0.0 but none is installed. You must install peer dependencies yourself.`

Presumably triggered by https://github.com/wikimedia/eslint-config-wikimedia/issues/248. Might be harmless but certainly confusing at the moment :)

**Comment by jdforrester** [comment]
It is harmless, yes. We're waiting for the VueJS plugin to be updated upstream.

**Comment by edg2s** [comment]
eslint-plugin-vue 7.0 seems to be close to release: https://github.com/vuejs/eslint-plugin-vue/releases

**Comment by edg2s** [comment]
cc @catrope, 7.0 has been released but I'll leave this to you to update

**Comment by jdforrester** [comment]
Fixed in #348.
